roles/deploy/hooks/finalize-before.yml

```diff
@@ -17,22 +17,17 @@
 
 - name: WordPress Installed (multisite)?
   block:
-    - name: Create file with multisite constants defined as false
-      copy:
-        src: "tmp_multisite_constants.php"
```

Comment:
This file can be deleted now too

Reply:
https://github.com/roots/trellis/pull/1391/commits/b58ca90e5d99617c713462952cc8c02904178557
This way to clean up the file by Ansible looks save enough I think (`file state: absent`).

Reply:
Sorry I meant https://github.com/roots/trellis/blob/17430191bb7211545eb63ba3ba989ee95c262c5f/roles/deploy/files/tmp_multisite_constants.php can be deleted 😅 

I'd be okay without the clean up task 🤷‍♂️ but it's fine either way

Reply:
@swalkinshaw: https://github.com/roots/trellis/pull/1391/files#diff-051d620484530c4b6b44daf807be7915a2c6b14046c3d0fd24f325b51ea353ad